fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1296,6 +1296,42 @@
     @ConfField
     public static long hive_meta_store_timeout_s = 10L;
 
+    /**
+     * If set to true, StarRocks will automatically synchronize hms metadata to the cache in fe.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_hms_events_incremental_sync = false;
+
+    /**
+     * HMS polling interval in milliseconds.
+     */
+    @ConfField(mutable = true)
+    public static int hms_events_polling_interval_ms = 5000;
+
+    /**
+     * Maximum number of events to poll in each RPC.
+     */
+    @ConfField(mutable = true)
+    public static int hms_events_batch_size_per_rpc = 500;
+
+    /**
+     * If set to true, StarRocks will process events in parallel.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_hms_parallel_process_evens = true;
```

Comment:
Can this configuration be removed? Setting hms_process_events_parallel_num to 1 means disable parallel processing.

Reply:
Ditto